src/features/presentation/components/webinaire/webinaire.component.html

```diff
@@ -1,7 +1,7 @@
 <div class="row row-cols-1 row-cols-md-3 g-4 justify-content-center">
   <div class="col" *ngFor="let webinaire of webinairePresentation">
     <div class="card h-100 rounded-0">
-      <img src="/assets/img/presentation/webinaire/{{ webinaire.image }}" class="w-100" alt="" />
+      <img src="/assets/img/presentation/webinaire/{{ webinaire.image }}" class="w-100" alt="'Webinaire ' + webinaire.titre" />
```

Comment:
```suggestion
      <img src="/assets/img/presentation/webinaire/{{ webinaire.image }}" class="w-100" alt="'" />
```
Ici par exemple l'information est déjà précisé avec le titre de la section `Découvrez nos webinaires, pour en savoir plus
sur cette cartographie et ses usages variés` et le titre du webinaire ligne 11.

L'image n'apporte aucune information supplémentaire. Mettre un texte alternatif dans ce cas est plutôt contre productif, car ça va être interprété par les outils d'accessibilité qui vont préciser une information déjà présente donc redondante.